xml/deployment_sysreqs.xml

```diff
@@ -781,6 +781,250 @@
   </para>
  </sect1>
 
+ <sect1 xml:id="sec.deploy.requirements.public_cloud">
+  <title>General Public Cloud Requirements</title>
+  <para>
+   In Amazon EC2, the adminstrative instance must be launched with an IAM role
+   that allows full access to the EC2 API. In Microsoft Azure, all security
+   credentials will be collected during setup. In Google Compute Engine, the
+   instance must be launched with an IAM role including <literal>Compute
+   Admin</literal> and <literal>Service Account Actor</literal> scopes.
+  </para>
+  <para>
+   Select an instance size for the Administrative node that meets the system
+   requirements as documented in the &productname; documentation.
+  </para>
+  <variablelist>
+   <varlistentry>
+    <term>Memory</term>
+    <listitem>
+     <para>
+      Minimal main memory: 8GB
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Volume Size</term>
+    <listitem>
+     <para>
+      Root volume size should be at least 40GB and is dependent on the size and
```

Comment:
This should go as a note into the "Minimum node specifications" the rest of this section can be dropped because it just duplicates the basic requirements.